presto-orc/src/test/java/com/facebook/presto/orc/TestMapFlatSelectiveStreamReader.java

```diff
@@ -602,19 +600,16 @@ private boolean passesFrequencyCheck(Frequency frequency, int i)
         }
     }
 
-    static class TestingFilterFunction
```

Comment:
Wouldn't that violate the "only 1 purpose per commit?" as this would mess with Masha's flatmap code

Reply:
To be clear because I think this is tricky -- there's a naming collision. What used to be called TestingFilterFunction is being renamed to TestingFlatMapFilterFunction. And now there's a new TestingFilterFunction. 

Separately, TestingFlatMapFilterFunction will inherit from the new TestingFilterFunction. 

The new TestingFilterFunction is in a new file, but I'll move it to a sub-interface of OrcTester. 

Does that work / am I understanding your nit correctly?

Reply:
Okay, trying it now...